src/main/kotlin/models/PeerDidResolverLocal.kt

```diff
@@ -111,9 +111,9 @@ object PeerDidResolverLocal {
         }
     }
 
-    private fun getVerificationMethod(did: String, decodedEncumbasis: DecodedEncumbasis) =
+    private fun getVerificationMethod(keyId:Int , did: String, decodedEncumbasis: DecodedEncumbasis) =
         VerificationMethodPeerDID(
-            id = "$did#${decodedEncumbasis.encnumbasis}",
+            id = "#key-${keyId}",
```

Comment:
```suggestion
            id = "$did#key-${keyId}",
```
let's keep the `did` just to be safe. I expect this might break other stuff